### PR TITLE
Add gentle explanations of new terms

### PR DESCRIPTION
**Pull request by rosiel**
Terms identified by the MIG Glossary as confusing, see https://docs.google.com/document/d/1cfPYFVV9qvvz2VjBRdYUN0CB7AyVDuG-GYavQ27DuBk/edit#

## Purpose / why

MIG noted some terms in the intro were confusing and required explanation. I tried to reword the introduction to be less jargon-y, and in a way that would explain the context/meaning behind those terms. 

## What changes were made?

(see diff)

## Verification

Does this track, and does this make more sense to folks who are new to the technical side?

## Interested Parties

@kspurgin and the Metadata Interest Group
---

## Checklist

> __Pull-request reviewer__ should ensure the following

* [ ] Does this PR link to related [issues](https://github.com/Islandora/documentation/issues/)?
* [ ] Does the proposed documentation align with the [Islandora Documentation Style Guide](https://islandora.github.io/documentation/contributing/docs_style_guide/)?
* [ ] Are the changes accurate, useful, free of typos, etc?

> __Person merging__ should ensure the following
* [ ] Does mkdocs still build successfully? (This is indicated by TravisCI passing. To test locally, and see warnings, see [How To Build Documentation](https://islandora.github.io/documentation/technical-documentation/docs-build/).)
* [ ] If pages are renamed or removed, have all internal links to those pages been fixed?
* [ ] If pages are added, have they been linked to or placed in the menu?
* [ ] Did the PR receive at least one approval from a committer, and all issues raised have been addressed?

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -30,11 +30,11 @@ Anyone can log in to this sandbox as an administrator and explore the interface!
...
+Islandora can be installed via an an [Ansible Playbook](https://github.com/Islandora-Devops/islandora-playbook), a set of instructions which provisions the full Islandora stack. It can be used to create a locally installed Islandora (requiring your computer to have Vagrant and Virtualbox) or can be used to provision a remote Linux server. The provisioning process involves many steps where software is downloaded and installed, so it can take a while. There is an option to get a basic ("standard") site, or to install a suite of demo configurations known as the Demo Install Profile. See [Installation - Ansible Playbook](installation/playbook) for more details.
```
Comment (seth-shaw-unlv) [inline docs/index.md]: You can take or leave the suggestion, but I would say "a machine-actionable set of instructions"

Comment (rosiel) [reply docs/index.md]: Thank you!

**Review by seth-shaw-unlv — APPROVED** [review]
👍
Looks good to me, but I'll leave it open in case anyone else wants to chime in.

**Review by kspurgin — APPROVED** [review]
Looks great to me. Thank you @rosiel

File: docs/index.md [hunk 2]
```diff
@@ -8,14 +8,14 @@ Islandora:
...
-- **Integrates with Fedora** - Drupal resources can be stored in Duraspace's [Fedora 5.x Repository](https://wiki.duraspace.org/display/FF/Fedora+Repository+Home) as binary assets and RDF metadata.
-- **Uses microservices** - Islandora provides an architecture for messaging and integration with any number of microservices, that provide services outside of the Drupal framework. Islandora's [Crayfish](https://github.com/Islandora/crayfish) suite of microservices provides functionality for synchronizing resources into Fedora and automatically generating derivative files.
-- **Can handle messages at scale** - Islandora uses [Alpaca](https://github.com/Islandora/Alpaca), an integration middleware based on Apache Camel, to scalably handle messaging and queueing for microservices such as Crayfish.
+- **Integrates with Fedora** - Drupal resources can be stored in Duraspace's [Fedora 5.x Repository](https://wiki.duraspace.org/display/FF/Fedora+Repository+Home) as binaries (files) and RDF metadata.
```
Comment (whikloj) [inline docs/index.md]: Duraspace is now Lyrasis and we support Fedora 6 as well. So maybe something like
```suggestion
- **Integrates with Fedora** - Drupal resources can be stored in Lyrasis's [Fedora Repository](https://wiki.duraspace.org/display/FF/Fedora+Repository+Home) (version 5.0 or greater) as binaries (files) and RDF metadata.
```

Comment (whikloj) [reply docs/index.md]: Otherwise I'm 👍 .

**Review by whikloj — APPROVED** [review]
I am good here and would merge, but as now I am listed as a co-author I thought it might make sense to get someone else to push the button.

**Pull request merged**